### PR TITLE
Update helloworld examples to be simple

### PR DESCRIPTION
**Pull request by erica-chiu**
**Related issues**
N/A

**Describe the proposed solution**
Add simple versions of the Iris and Boston examples.

**Describe alternatives you've considered**
N/A

**Additional context**

### CODE REVIEW

File: docs/examples/Titanic-Binary-Classification.md [hunk 1]
```diff
@@ -36,13 +36,13 @@ In the main function, we create a spark session as per usual:
...
+We then define the set of raw features that we would like to extract from the data. The raw features are defined using [FeatureBuilders](../developer-guide#featurebuilders), and are strongly typed. TransmogrifAI supports the following basic feature types: Text, Numeric, Vector, List , Set, Map. In addition it supports many specific feature types which extend these base types: Email extends Text; Integral, Real and Binary extend Numeric; Currency and Percentage extend Real. For a complete view of the types supported see the [Type Hierarchy and Automatic Feature Engineering](../developer-guide#type-hierarchy-and-automatic-feature-engineering) section in the Documentation.
```
Comment (tovbinm) [inline docs/examples/Titanic-Binary-Classification.md]: I dont think `../` prefixes would work once we deploy to https://docs.transmogrif.ai
were the links broken? how did you test them?

Comment (erica-chiu) [reply docs/examples/Titanic-Binary-Classification.md]: The links are broken on the website, but only for that paragraph. I just imitated the style of the links that work in the other paragraphs. How do I link a page that will work when deployed?

Comment (tovbinm) [reply docs/examples/Titanic-Binary-Classification.md]: Try running the docs server locally as described here - https://github.com/salesforce/TransmogrifAI/tree/master/docs#docs

Comment (tovbinm) [reply docs/examples/Titanic-Binary-Classification.md]: Oh, you're right. We actually have them as `../` elsewhere.

Comment (erica-chiu) [reply docs/examples/Titanic-Binary-Classification.md]: They work.

**Pull request merged**